packages/ui/exports/define/lion-uploaded-file-list.js

```diff
@@ -0,0 +1,3 @@
+import { LionSelectedFileList } from '../input-file.js';
+
+customElements.define('lion-uploaded-file-list', LionSelectedFileList);
```

Comment:
```suggestion
customElements.define('lion-selected-file-list', LionSelectedFileList);
```

And we should also update this file name.